server/models/Order.js

```diff
@@ -446,6 +447,48 @@ function defineModel() {
     });
   };
 
+  /**
+   * Cancels all subscription orders in the given tier
+   */
+  Order.cancelActiveOrdersByTierId = function (tierId) {
+    return Order.update(
+      { status: OrderStatuses.CANCELLED },
+      {
+        where: {
+          TierId: tierId,
+          SubscriptionId: { [Op.not]: null },
+          status: {
+            [Op.not]: [OrderStatuses.PAID, OrderStatuses.CANCELLED, OrderStatuses.REJECTED, OrderStatuses.EXPIRED],
+          },
+        },
+      },
+    );
+  };
+
+  /**
+   * Cancels all orders with subscriptions that cannot be transferred when changing hosts (i.e. PayPal)
+   */
+  Order.cancelNonTransferableActiveOrdersByCollectiveId = function (collectiveId) {
+    return sequelize.query(
+      `
+        UPDATE public."Orders" SET status = 'CANCELLED'
+        WHERE id IN (
+          SELECT "Orders".id FROM public."Orders"
+          INNER JOIN public."Subscriptions" ON "Subscriptions".id = "Orders"."SubscriptionId"
+          WHERE
+            "Orders".status NOT IN ('PAID', 'CANCELLED', 'REJECTED', 'EXPIRED') AND
+            "Subscriptions"."isManagedExternally" AND
+            "Subscriptions"."isActive" AND
+            "Orders"."CollectiveId" = ?
+        )
+      `,
+      {
+        type: QueryTypes.UPDATE,
+        replacements: [collectiveId],
```

Comment:
:ice_cream: We usually prefer to use named parameters as they're easier to deal with than positional, but no required change here.